assets/tests/components/propertiesPanel/vehiclePropertiesPanel.test.tsx

```diff
@@ -186,13 +187,15 @@ describe("VehiclePropertiesPanel", () => {
 
   test("Includes invalid bus banner when vehicle is off course", () => {
     render(
-      <VehiclePropertiesPanel
-        selectedVehicle={invalidVehicleFactory.build()}
-        tabMode="status"
-        onChangeTabMode={jest.fn()}
-        onClosePanel={jest.fn()}
-        openMapEnabled={true}
-      />
+      <MemoryRouter initialEntries={["/"]}>
```

Comment:
question(non-blocking): just wondering, do you know why these tests needed a `<MemoryRouter/>` added?

Reply:
It's because of the `<Link>` in the minimap for the "Open Map" button. Previously, that was only rendered if some of the test groups were enabled so it wasn't an issue, but now it's always rendered.